libafl/src/executors/forkserver.rs

```diff
@@ -830,6 +852,20 @@ impl<'a> ForkserverExecutorBuilder<'a, StdShMemProvider> {
         self
     }
 
+    #[must_use]
+    /// Call this if you want to run it under persistent mode; default is false
```

Comment:
We should default to true, there's never a reason not to set that env I think?

Reply:
because non-persistent binary does not work if it's true.
I think it's better to set it to false so both non-persistent and persistent one work

Reply:
It's a footgun, though